Source/JavaScriptCore/wasm/WasmFaultSignalHandler.cpp

```diff
@@ -96,7 +114,11 @@ static SignalAction trapHandler(Signal signal, SigInfo& sigInfo, PlatformRegiste
             };
 
             if (didFaultInWasm(faultingInstruction)) {
+#if defined(EXCEPTION_STATE_IDENTITY_PROTECTED) && !PLATFORM(IOS_FAMILY_SIMULATOR)
```

Comment:
Should this be `HAVE(HARDENED_MACH_EXCEPTIONS)` or at least include `CPU(ARM64E)`?